routes/story.route.server.js

```diff
@@ -136,6 +137,30 @@ module.exports = app => {
             });
     };
 
+    addRatingToStory = (req, res) => {
+        if (!ObjectID.isValid(req.body.storyID)) {
+            return res.status(404).send();
+          }
+
+        if(req.body.role) {
+            if(req.body.role == role.MODERATOR || req.body.role == role.ADMIN) {
+                storyDao.findStoryByStoryID(req.body.storyID)
```

Comment:
Here it's better to use. === than == because of best practise

Reply:
yeah the strict equality is for strings, but now I'm comparing primitive numbers and not objects

Reply:
=== for matching value and type and == just compares values

Reply:
Kanika had used == in last PR but Vishal changed it to ===
I had asked the same doubt and it was decided that we won't allow deletion if front end doesn't send the right type